hooks/checkout

```diff
@@ -46,10 +46,14 @@ setup_git_repo() {
   local CHECKOUT_REF="$2"
   local SSH_KEY_PATH="$3"
   local CLONE_DIR="$4"
+  local DEPTH_FLAG="$5"
 
   echo " :::: local clone location = '$CLONE_DIR'"
   echo " :::: git remote url = '$REPO_URL'"
   echo " :::: ssh key path = '$SSH_KEY_PATH'"
+  if [[ -z "${DEPTH_FLAG}" ]]; then
```

Comment:
Doesn't `-z` return true if the string is empty? I feel like this should be `-n` unless I'm missing something

reference https://stackoverflow.com/questions/18096670/what-does-z-mean-in-bash

Reply:
Whoops, you're right.